tests/metrics/functional/test_sequence.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+import torch
+from nltk.translate.bleu_score import corpus_bleu, sentence_bleu
+
+from pytorch_lightning.metrics.functional.sequence import bleu_score
+
+# example taken from
+# https://www.nltk.org/api/nltk.translate.html?highlight=bleu%20score#nltk.translate.bleu_score.sentence_bleu
+hypothesis1 = "It is a guide to action which ensures that the military always obeys the commands of the party".split()
```

Comment:
As they are constants, pls use CAPITALS in name

Reply:
Changes Done